plugin/mutate/source/dextool/plugin/mutate/backend/database/standalone.d

```diff
@@ -280,12 +280,50 @@ struct Database {
         return app.data.sort!((a, b) => a.timeStamp < b.timeStamp).array;
     }
 
+    /// Returns: the stored scores in ascending order by their `time`.
+    FileScore[] getMutationFileScoreHistory() @trusted {
```

Comment:
instead of retrieving all it seems like it is only those with a timestamp > 7days that are of interest.
i recommend adding that as a parameter, modify the query and let SQL sort them by date. then most D logic and cleanup are captured by the query